commander/src/commander-plugin.c

```diff
@@ -32,18 +32,8 @@
 /*#define DISPLAY_SCORE 1*/
 
 
-GeanyPlugin      *geany_plugin;
-GeanyData        *geany_data;
-
-PLUGIN_VERSION_CHECK(226)
-
-PLUGIN_SET_TRANSLATABLE_INFO (
-  LOCALEDIR, GETTEXT_PACKAGE,
-  _("Commander"),
-  _("Provides a command panel for quick access to actions, files and more"),
-  VERSION,
-  "Colomban Wendling <ban@herbesfolles.org>"
-)
+static GeanyPlugin      *geany_plugin = NULL;
+static GeanyData        *geany_data = NULL;
```

Comment:
These globals should be removed then, unless it's *highly* impractical

Reply:
How? Move them into ```struct plugin_data``` or try to pass them as user pointers, e.g. from ```plugin_commander_init()``` to ```on_plugin_idle_init (-->create_panel...)```.